<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RolePageForAp</class>
 <widget class="QWidget" name="RolePageForAp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="rolePageBg" native="true">
     <property name="styleSheet">
      <string notr="true">#rolePageBg{
	border-radius: 10px;
	background-color: white;
}


*{
	font-family: ‘微软雅黑’;
	font-size: 14px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="inputBtnArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#inputBtnArea{
	border-radius : 10px;
	
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>41</number>
         </property>
         <property name="leftMargin">
          <number>23</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>27</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="rolePhoneLabel" native="true">
           <property name="minimumSize">
            <size>
             <width>345</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>345</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
#rolePhoneLabel{
color:black;

border-radius : 4px;
border: none;
/* border : 1px solid #DCDEE0; */
padding-left : 18px;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color:black;</string>
              </property>
              <property name="text">
               <string>用户手机号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="rolePhoneEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>大陆86:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="roleStatuBox" native="true">
           <property name="minimumSize">
            <size>
             <width>297</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>297</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="roleStatuLabel">
              <property name="minimumSize">
               <size>
                <width>74</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>74</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color:black;</string>
              </property>
              <property name="text">
               <string>用户状态</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="roleStatusCombox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#roleStatusCombox {
background-color: #FFFFFF;
border: 1px solid #DCDEE0;
border-radius: 4px;
color: #222222;
padding-left: 16px;
line-height: 40px;
}
#roleStatusCombox::drop-down {
border: none;
width: 33px;
}
#roleStatusCombox::down-arrow {
width: 15px;
background-color: transparent;
image: url(:/image/more.png);
}
#roleStatusCombox QAbstractItemView {
outline: 0px;
}
#roleStatusCombox QAbstractItemView::item {
color: #222222;
padding-left:16px;
background-color: #FFFFFF;
height: 30px;
}
#roleStatusCombox QAbstractItemView::item:selected {
background-color: #409CE1;
color: #FFFFFF;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>210</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="btnBox" native="true">
           <property name="minimumSize">
            <size>
             <width>194</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>194</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="resetBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#resetBtn{
	color: #222222;
	background-color: #FFFFFF;
	border: 1px solod #DEDEE0;
	border-radius: 4px;
}</string>
              </property>
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="queryBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#queryBtn{
	color: #FFFFFF;
	background-color: #3ECEFF;
	border-radius: 4px;
	border: 1px solod #DEDEE0;
}</string>
              </property>
              <property name="text">
               <string>查询</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="addRoleBtnArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>74</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>74</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
color:black ;</string>
        </property>
        <widget class="QLabel" name="addImage">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>28</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#addImage{
	border-image: url(&quot;:/image/addrole.png&quot;);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="addRoleBtn">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>19</y>
           <width>100</width>
           <height>40</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#addRoleBtn{
	border: none;
	color: #3ECEFF;

	border-radius: 8px;
	border: 1px solid #3ECEFF;
	padding: 10px 25px 10px 27px;
}</string>
         </property>
         <property name="text">
          <string>新增</string>
         </property>
        </widget>
        <zorder>addRoleBtn</zorder>
        <zorder>addImage</zorder>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="tableTitleArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>74</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#tableTitleArea{

	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
}

</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="idLabel">
           <property name="minimumSize">
            <size>
             <width>174</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>174</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#idLabel{


	font-weight: bold;
	color: #222222;
	line-height: 20px;
	background-color: #EBFAFF;

}</string>
           </property>
           <property name="text">
            <string>序号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="roleLabel">
           <property name="minimumSize">
            <size>
             <width>174</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>174</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#roleLabel{
	font-weight: bold;
	color: #222222;
	line-height: 20px;
	background-color: #EBFAFF;

}</string>
           </property>
           <property name="text">
            <string>用户角色</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="phoneLabel">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#phoneLabel{
	font-weight: bold;
	color: #222222;
	line-height: 20px;
	background-color: #EBFAFF;

}</string>
           </property>
           <property name="text">
            <string>手机号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#nameLabel{
	font-weight: bold;
	color: #222222;
	line-height: 20px;
	background-color: #EBFAFF;

}</string>
           </property>
           <property name="text">
            <string>用户昵称</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="statuLabel">
           <property name="minimumSize">
            <size>
             <width>188</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>188</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#statuLabel{
	font-weight: bold;
	color: #222222;
	line-height: 20px;
	background-color: #EBFAFF;

}</string>
           </property>
           <property name="text">
            <string>状态</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noteLabel">
           <property name="minimumSize">
            <size>
             <width>188</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>188</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#noteLabel{
	font-weight: bold;
	color: #222222;
	line-height: 20px;
	background-color: #EBFAFF;

}</string>
           </property>
           <property name="text">
            <string>备注</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="opeLabel">
           <property name="styleSheet">
            <string notr="true">#opeLabel{
	font-weight: bold;
	color: #222222;
	line-height: 20px;
	background-color: #EBFAFF;

}</string>
           </property>
           <property name="text">
            <string>操作</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="roleInfoArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>408</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>408</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">#scrollArea{
	border: none;
	background-color: rgb(163, 163, 163);
}

QScrollBar:vertical
{
	border:none;
	background-color: #FFFFFF;	
	width: 10px;
}



QScrollBar::handle:vertical{
	border: none;
	border-radius: 5px;
	background-color: #E8E8E8;
	min-height: 134px;
}

QScrollBar::up-arrow:vertical,
QScrollBar::down-arrow:vertical
{
	border: none;
	width: 0px;
	height: 0px;
}


</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1270</width>
              <height>408</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="layout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>11</number>
               </property>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="paginatorArea" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
